lib/default.nix

```diff
@@ -1,4 +1,16 @@
 { lib }:
+let
+  fsTypeToUtil = pkgs: fs: with pkgs;{
```

Comment:
The `pkgs` here is `buildPackages`? Otherwise, we could use the global instance.

Reply:
There's a few ways around this
1. We keep this
2. We can make vmHostPackages an extra parameter to lib/default.nix and pass it in. But this gets messy (I tried) because there are quite a few call sites, and not all of them require vmHostPackages
3. We could just set vmHostPackages with a default value of the `host.cpu == null ...` check, and access it from `config` whenever we need it. I think this is the cleanest solution.